### DISCUSSION TITLE
[Authentication] Add support for Apple

### DISCUSSION THREAD

**Discussion by nickrandolph** [unoplatform/uno.extensions#1885] [Ideas]
## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

**Comment by MartinZikmund** [comment]
We now have a community-based sample of such implementation https://github.com/unoplatform/uno/pull/18410, it would be great to include this as an option in Auth
